js/app/dashboard/components/MapProxy.js

```diff
@@ -250,9 +247,13 @@ export default class MapProxy {
       if (!marker.getLatLng().equals(latLng)) {
         marker.setLatLng(latLng).update()
       }
+    }
 
-      popupComponent.current.updateTask(task)
+    popupComponent = this.taskPopups.get(task['@id'])
 
+    if (popupComponent) {
+      console.debug(`Popup for Task #${task['@id']} already exists, updating...`)
```

Comment:
Do you need it on production?

Reply:
I dont need in production it is why i set it to "debug"

I guess it will be sent to console anyway, but I don't know what are good JS practice here for logger configuration?